app/models/container_image.rb

```diff
@@ -31,9 +31,14 @@ class ContainerImage < ApplicationRecord
 
   acts_as_miq_taggable
   virtual_column :display_registry, :type => :string
+  virtual_column :number_of_containers, :type => :integer
```

Comment:
naming: `container_count`?

@zeari @cben what does this actually mean? the number of running containers?
Does that include archived containers? (we probably don't want those)